graders/python/python_autograder/code_feedback.py

```diff
@@ -299,6 +299,59 @@ def bad(msg):
 
         return True
 
+    @classmethod
+    def check_dict(
+        cls,
+        name,
+        ref,
+        data,
+        entry_type=None,
+        accuracy_critical=False,
+        report_failure=True,
+    ):
+        """
+        Feedback.check_dict(name, ref, data)
+
+        Check that a student dict has correct length with respect to a reference dict.  Can also check for a homogeneous data type for the list.
```

Comment:
It says that only the length is checked for, but it looks like this function checks for equality between the dicts.